src/pages/layer-2.js

```diff
@@ -230,10 +230,8 @@ const Layer2Page = ({ data }) => {
           "https://api.cryptostats.community/api/v1/l2-fees/feeTransferEth?metadata=false"
         )
 
-        // Temporary, but filtering out L2's we arent listing
-        feeData = feeData.data.filter(
-          (l2) => l2.id !== "aztec-network" && l2.id !== "hermez"
-        )
+        // Filter out layer 2 networks not listed
+        feeData = feeData.data.filter((l2) => l2.id !== "hermez")
```

Comment:
We should probably explicitly get the fee data we want here instead of filtering out what we don't want. E.g. Metis was recently added to this list, which I believe we don't intend on listing on the layer 2 page.

Reply:
Good call, will make an issue for this.